### DISCUSSION TITLE
Comparison chart

### DISCUSSION THREAD

**Discussion by craciu25yt** [yhype/yhype-feedback#23] [Ideas]
Show a chart that for example shows the views, clones, stars, forks from all your repos or if you have to many the last 20 or something like this in the last 7 days

**Comment by antonkomarev** [comment]
Interesting idea.

The hardest part — decide where this data should be displayed (on what page).

You think it's good to see a comparison of repositories over one profile/organization? Or compare repos between organizations as well?

**Comment by andrey-helldar** [comment]
I too miss the overall summary of all repositories in the organization.

I think you can do the same as on the "dashboard" page by adding one tab:

- Dashboard
- Views
- Followers
- Sponsors
- Repositories < that one

**Reply by antonkomarev** [reply]
Good idea, will try to find time for this feature. I think it will be good to pick from any repositories you are a member. But we should limit repos count for the summary.

**Reply by antonkomarev** [reply]
@andrey-helldar @craciu25yt I've added raw experimental page, where you can see unique visitors count comparison between your repos. This feature should be revised later, just a mockup to test how it works. Will be glad to receive ideas and feedback.

https://yhype.me/github/profile-repositories

**Reply by andrey-helldar** [reply]
@antonkomarev, by default, all checkboxes are off, and the list contains all repositories.

I can suggest filtering this list by excluding repositories deactivated in the settings.

**Reply by andrey-helldar** [reply]
I also noticed that the filter settings are saved as URL parameters.

That is, if you go to another page and then visit this page again, you will have to set all the checkboxes manually again, since their state is not saved either on the backend or on the frontend in localStorage, at least.

But as an MVP it's quite a good statistic :)

**Reply by antonkomarev** [reply]
@andrey-helldar Yeah, that was a concept, how it should work in MVP. Just take all repos where the user has access to. But I like the idea to show only enabled to parse stat repos. But it might be not obvious. Will require some text notes.

I haven't added any kind of save feature to make it faster to implement without any database design.

I want to think about this feature more before doing the save button. Ideally I want to be able to compare all stats. Views, visitors, followers, etc.

**Reply by antonkomarev** [reply]
@andrey-helldar Now it should display only repos with enabled traffic collector.

**Reply by andrey-helldar** [reply]
@antonkomarev, Thank you! That's so much better :)
